### ISSUE TITLE
aarch64: virtual address space underrun!

### ISSUE THREAD

**Issue by prusnak** [NixOS/patchelf#208]
**Describe the bug**

patchelf fails with error "virtual address space underrun! " while patching the Electron 3 binary for aarch64.
See https://hydra.nixos.org/build/120910216

**Steps To Reproduce - Option 1**

Try to build electron_3 on nixpkgs master (on aarch64).

**Steps To Reproduce - Option 2**

Run the following script (on aarch64) with patchelf or patchelfUnstable installed:

```
vers=3.1.13
tag=linux-arm64
wget -nc https://github.com/electron/electron/releases/download/v${vers}/electron-v${vers}-${tag}.zip
unzip electron-v${vers}-${tag}.zip
patchelf electron
```

**Comment by prusnak** [comment]
Not sure if this is not already fixed on patchelf master, but nixpkgs still use older version (
The patchelf issue will be probably fixed by patchelf 0.11 https://github.com/NixOS/nixpkgs/issues/69213)

**Comment by domenkozar** [comment]
That's a duplicate of #80
